### ISSUE TITLE
Propagate the Rendered Dimensions

### ISSUE THREAD

**Issue by jd-carroll** [Famous/engine#321]
Currently, the width and height of the rendered node are not available without querying the DOM.  If you set the proportional size of a node, on the next frame the actual rendered size should be available on that node.

**Comment by michaelobriena** [comment]
It is possible to get the size of the node from the getSize api on the node.  The only caveat is that relative size and rendered size need to be called asynchronously since we can't know what the scene containers size is nor the size of a rendered element before we draw it.

```js
'use strict';

// Famous dependencies
var DOMElement = require('famous/dom-renderables/DOMElement');
var FamousEngine = require('famous/core/FamousEngine');

// Boilerplate code to make your life easier
FamousEngine.init();

// Initialize with a scene; then, add a 'node' to the scene root
var logo = FamousEngine.createScene().addChild();

// Create an [image] DOM element providing the logo 'node' with the 'src' path
new DOMElement(logo, { tagName: 'img' })
    .setAttribute('src', './images/famous_logo.png');

// Chainable API
logo
    // Set size mode to 'absolute' to use absolute pixel values: (width 250px, height 250px)
    .setSizeMode('relative', 'absolute', 'absolute')
    .setAbsoluteSize(250, 250)
    .setProportionalSize(.4, .4)
    // Center the 'node' to the parent (the screen, in this instance)
    .setAlign(0.5, 0.5)
    // Set the translational origin to the center of the 'node'
    .setMountPoint(0.5, 0.5)
    // Set the rotational origin to the center of the 'node'
    .setOrigin(0.5, 0.5);

// Add a spinner component to the logo 'node' that is called, every frame
var spinner = logo.addComponent({
    onUpdate: function(time) {
        logo.setRotation(0, time / 1000, 0);
        logo.requestUpdateOnNextTick(spinner);
    }
});

// Let the magic begin...
logo.requestUpdate(spinner);

setTimeout(function() {
    console.log(logo.getSize())
}, 2000);
```

**Comment by jd-carroll** [comment]
Does that example work for you?  Because it's not working for me, it produces strange results where:
- the logo will not respect the alignment [always thinks it is at `.setAlign(x, 0)`]
- it will stop setting the hieght for the logo if I start messing with the sizeMode

**Comment by michaelobriena** [comment]
Yes, that example is working for me?  What commit are you on?

Can you send your example code as well?

**Comment by jd-carroll** [comment]
I was using that code on 0.5.x (the same exact code above) and it was behaving all strange.. 

Now that you have actually published 0.6.2, its working without issue.  This includes the original issue where the size was not being populated.
